examples/synchronization-mutex-tmpl-level.yaml

```diff
@@ -0,0 +1,45 @@
+# This example demonstrates the use of a Synchronization Mutex lock on template execution. Mutex lock limits
+# only one of the template execution across the workflows in the namespace which has same Mutex lock.
```

Comment:
how do a create a lock across namespaces?

Reply:
Mutex and Semaphore features are namespaced scope.

Reply:
I can see that people might want this is v2, as there are plenty of non-Kuberenetes resources that are not namespace scoped.  My thought would be that we just add `namespace` as an argument to the YAML. But not today.